protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -176,6 +195,30 @@ definitions:
         type: array
         items:
           "$ref": "#/definitions/AirbyteStreamState"
+  AirbyteFileType:
```

Comment:
nit: What is `Type` referring to in this name? When I hear "type", I think about an enum like `failure_type` [here](https://github.com/airbytehq/airbyte-protocol/blob/009278cfa8677e1bfdfaeb7cabfc8415c06aef34/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml#L313-L319).

Reply:
It's fair, I think i can reuse the idea of a descriptor which makes more sense, then we can have `AirbyteFileDescriptor`

Reply:
Would this be a breaking change considering the current implementation?

Reply:
> Would this be a breaking change considering the current implementation?

Do you mean this ref for this comment? No

Introducing the new message to protocol? It depends, it would be replacing [this message in cdk](https://github.com/airbytehq/airbyte-python-cdk/blob/main/airbyte_cdk/models/file_transfer_record_message.py).

Renaming keys or changing expected structure could lead to changes on the cdk (not a big problem) and platform/destination (more problematic) , I think it wouldn't be a change to impact on connectors but may we depend on coordinate to introduce such changes.